install_files/STDCMenuCreator_v1.ms

```diff
@@ -0,0 +1,55 @@
+-- Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+-- Compatibility: 3ds Max 2025+
+
+-- Function to create the AWS Deadline menu
+function createDeadlineCloudMenu =
+(
+    -- Get the menu manager from the callback parameter
+    local menuMgr = callbacks.notificationParam()
+    local mainMenuBar = menuMgr.mainMenuBar
+    
+    -- No need to check for existing menu - the CreateSubMenu will handle it
+    -- If a menu with the same GUID already exists, it will be replaced
+    
+    -- Create a new AWS Deadline menu with a unique GUID
+    -- Using a fixed GUID ensures consistency across sessions
+    local deadlineCloudMenuGUID = "F8FFB827-741C-4A81-8C89-BBF856DCF56D"
+    local deadlineCloudMenu = mainMenuBar.CreateSubMenu deadlineCloudMenuGUID "AWS Deadline"
+    
+    -- Define menu items with their GUIDs
+    local menuItems = #()
+    
+    -- Add the Submit to Deadline Cloud menu item
+    local submitActionGUID = "6AC5FDFE-2AF7-4069-8493-D8A1462F4E7F"
+    -- MacroScript action table ID is 647394
+    local macroScriptActionTableID = 647394
+    
+    -- Create the Submit to Deadline Cloud action
+    deadlineCloudMenu.CreateAction submitActionGUID macroScriptActionTableID "SubmitToDeadlineCloud`DeadlineCloud" title:"Submit to Deadline Cloud"
+    
+    -- Add the Test Job Bundle Output menu item if in DEBUG mode
+    local DEBUG = false
```

Comment:
nit: Could we put this in the top of the file with a comment similar to the v0? It looks like we need to manually set this to true to get these debug menus?

```
	-- change to true for development install
	local DEBUG = false
```